webui/react/Makefile

```diff
@@ -1,6 +1,10 @@
 swagger_client = "src/services/swagger"
 PROJECT_ROOT := "../../"
 
+.PHONY: all
+all: get-deps
+	$(MAKE) build
```

Comment:
I would expect `all` to include `build-storybook`. I would either add that to `all`, or just rename this to something like `get-deps-and-build` (since the long name won't matter if you are just running `make`)

Reply:
aah I see your point. From what I see we use `all` to build what's needed for a module at each level.  in this case, `build-storybook` is just used as a test in our setup. we don't publish it anywhere nor do we use it. that target is called in CI just to make sure it passes. wdy think? we could totally add that in there but stroybook be a "needed" target.
get-deps-and-build makes sense. (or maybe default) ? :thinking:

Reply:
That makes sense. I would probably rename the make target to `default`, but `all` seems fine too if that's what you want to do with.